picard/coverart/image.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, data, prefix='picard', suffix=''):
     def __eq__(self, other):
         return self._hash == other._hash
 
+    def hash(self):
```

Comment:
Because IMHO that function doesn't return the hash of that object, but the hash of the CoverArtImage stored in that DataHash object.